### ISSUE TITLE
🐛 BUG: CSS gets added to the head in reverse order

### ISSUE THREAD

**Issue by kossnocorp** [snowpackjs/astro#1260]
### What package manager are you using?

npm

### What operating system are you using?

Windows

### Describe the Bug

I use `astro@0.20.3`. When I import CSS files, they get added on the page in reverse order which breaks the specificity order:

![image](https://user-images.githubusercontent.com/52201/131237837-55b316c9-c4b1-41fd-9cf4-5d1595c30115.png)

When I put CSS reset import first in the layout and then import global styles that define box mode, font family, etc., reset ends up being the last that breaks the styles.

### Steps to Reproduce

1. Using any Astro template.
2. Create a layout; import CSS in order: reset, global styles (i.e., defining the box model, font family, etc.)
3. Create a page that renders the layout; import page-specific CSS.
4. CSS will break as reset is added the last.

**Comment by FredKSchott** [comment]
@natemoo-re this would be a good one to tackle in the new compiler.

**Comment by natemoo-re** [comment]
I think we have the same bug in the new compiler, but I just opened a PR to fix this. https://github.com/snowpackjs/astro/pull/1721

**Comment by natemoo-re** [comment]
Update: #1721 was not the correct approach! I opened https://github.com/snowpackjs/astro-compiler-next/pull/130 on the compiler to fix this.

**Comment by natemoo-re** [comment]
This has been implemented and is available in `astro@next`!

---

The team has decided to close out issues that have been confirmed as fixed by [astro@0.21.0-next.0](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.0), [astro@0.21.0-next.1](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.1), or [astro@0.21.0-next.2](https://github.com/snowpackjs/astro/releases/tag/astro%400.21.0-next.2). Our hope is that this will help the [v0.21 milestone](https://github.com/snowpackjs/astro/milestone/1) remain as actionable as possible.

To verify that this issue has been fixed, you may
- Open a new [`astro@next` project on StackBlitz](https://astro.new/minimal@next)
- Locally upgrade to the `astro@next` preview
    ```shell
    npm i astro@next
    # If using Framework renderers...
    npm i @astrojs/renderer-name@next
    ```

If you think we made a mistake, **please reopen this issue**!

If you run into any other problems with `astro@next`, please [open a new issue](https://github.com/snowpackjs/astro/issues/new?assignees=&labels=0.21&template=---01-bug-report.yml&title=%F0%9F%90%9B+BUG%3Aastro@next).
